pom.xml

```diff
@@ -86,9 +86,9 @@
 			</dependency>
 
 			<dependency>
-				<groupId>org.hibernate</groupId>
+				<groupId>org.hibernate.validator</groupId>
 				<artifactId>hibernate-validator</artifactId>
-				<version>5.4.3.Final</version>
+				<version>6.1.6.Final</version>
```

Comment:
we should wait longer to merge this, since we still rely on javax.el 2.0.0 and hibernate-validator 6 depends on javax.el 3.0.0

Anyway, billy should not depend on hibernate-validator, which is an implementation, but on javax.validation which is the API, and leave to applications to choose which implementations to use.

Reply:
> Anyway, billy should not depend on hibernate-validator, which is an implementation, but on javax.validation which is the API, and leave to applications to choose which implementations to use.

done in https://github.com/premium-minds/billy/pull/147